x-pack/packages/ml/trained_models_utils/index.ts

```diff
@@ -27,4 +27,9 @@ export {
   ELASTIC_MODEL_TYPE,
   MODEL_STATE,
   type ModelState,
+  ELSER_LINUX_OPTIMIZED_MODEL_ID,
```

Comment:
Did you consider adding a method to [models_provider](https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/models/model_management/models_provider.ts#L440) instead of static constants? 
More likely we will introduce more models, and IDs might change, e.g. `.elser_model_2` used to be `.elser_model_2_SNAPSHOT` during the dev process.

Reply:
I did, but that would require moving a lot of this code to the server which would add even more scope to this already very big PR. I'll add a util function to the frontend to centralize this, that probably makes more sense.